<?xml version="1.0" encoding="utf-8"?>
<launch>

	<arg name="network_param_file" default="$(find city_simulation)/config/yolo_v3_tiny.yaml" />
	<arg name="image" default="/onboard_camera/image_raw" />

	<include file="$(find darknet_ros)/launch/darknet_ros.launch">
		<arg name="network_param_file" value="$(arg network_param_file)" />
		<arg name="image" value="$(arg image)" />
	</include>

</launch>